www/include/configuration/configObject/traps/formTraps.php

```diff
@@ -381,19 +422,24 @@ function myReplace()
 }
 $tpl->assign("helptext", $helptext);
 
-if ($o == "w") {
+if ($o == TRAP_WATCH) {
     # Just watch a Command information
     if ($centreon->user->access->page($p) != 2) {
-        $form->addElement("button", "change", _("Modify"), array("onClick"=>"javascript:window.location.href='?p=".$p."&o=c&traps_id=".$traps_id."'"));
+        $form->addElement(
+            "button",
+            "change",
+            _("Modify"),
+            array("onClick"=>"javascript:window.location.href='?p=".$p."&o=c&traps_id=".$traps_id."'")
```

Comment:
Add whitespace before & after the key, value, and on the concatenation
```suggestion
             array("onClick" => "javascript:window.location.href='?p=" . $p . "&o=c&traps_id=" . $traps_id . "'")
```